webanno-ui-annotation/src/main/java/de/tudarmstadt/ukp/clarin/webanno/ui/annotation/dialog/OpenDocumentDialogPanel.java

```diff
@@ -115,14 +124,20 @@ public OpenDocumentDialogPanel(String aId, AnnotatorState aBModel, ModalWindow a
             projectSelectionForm.setVisible(false);
         }
         
+        isAdmin = userRepository.isAdministrator(currentUser);
```

Comment:
Do not cache the value - fetch it when necessary. Also, use `ProjectService.isManager(Project, User)` instead of `userRepository.isAdministrator(currentUser)`.